salt/kubelet/init.sls

```diff
@@ -116,6 +120,24 @@ remove-should-uncordon-grain:
     - name: kubelet:should_uncordon
     - destructive: True
 
+# TODO: onlyif's need doing
+
+{% if "kube-master" in salt['grains.get']('roles', []) %}
```

Comment:
Ah. So - This will fail given kube-api isnt running - it only works in dev as I had already bootstrapped. Humm.